lib/src/errors.dart

```diff
@@ -99,16 +99,16 @@ class MultiError extends Error {
   /// The order of values is preserved, and if any error occurs, the
   /// [MultiError.errors] list will have errors in the corresponding slots,
   /// and `null` for non-errors.
-  Future<List<Object>> wait<T>(Iterable<Future<T>> futures,
-      {void Function(T successResult) cleanUp}) {
-    Completer<List<Object>> completer;
+  Future<List<Object?>> wait<T>(Iterable<Future<T>> futures,
+      {void Function(T successResult)? cleanUp}) {
+    late Completer<List<Object?>> completer;
```

Comment:
Again, would just initialize with the completer immediately.
It is an unnecessary overhead *only* if the iterable is empty, and I can live with that.